src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockGoogleCloudStorageDAO.scala

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import java.util.UUID
+
+// TODO flesh this out
```

Comment:
It looks like Ted fleshed it out a bit after I put in this TODO, so I'm guessing it can be removed